src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -1257,7 +1257,10 @@ public static bool SupportsINuGetProjectSystem(EnvDTEProject envDTEProject)
         /// </summary>
         public static async Task<bool> HasBuildIntegratedConfig(EnvDTEProject project)
         {
-            return await ContainsFile(project, BuildIntegratedProjectUtility.ProjectConfigFileName);
+            var projectNameConfig = BuildIntegratedProjectUtility.GetProjectConfigWithProjectName(project.Name);
+
+            return await ContainsFile(project, BuildIntegratedProjectUtility.ProjectConfigFileName)
```

Comment:
DTE could always throw, but if an exception is throw we don't have enough context to handle it and really the entire operation needs to stop since we won't be able to determine the project type.

This method is used to determine if a project is a non-packages.config project so that it can skip over the code for those scenarios.